cfgov/v1/wagtail_hooks.py

```diff
@@ -43,6 +43,17 @@ def share_the_page(request, page):
         flush_akamai(page)
 
 
+@hooks.register('after_delete_page')
+def log_page_deletion(request, page):
+    logger.warning(
```

Comment:
This looks good, but I'm not sure it'll be [caught wit our production logging settings](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/production.py#L5).